example1.ipynb

```diff
@@ -26,7 +26,10 @@
    "cell_type": "markdown",
```

Comment:
<p>We can comment on individual cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/brian-rose/notebook_diff_tutorial/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>